### PR TITLE
[WIP] Andreeva Elina Tech Writer Test

### PR DESCRIPTION
**Pull request by ElinaAndreeva**
### Motivation and context
I have made some little changes to made the documentation more complete. 
Also, I want to say that the existing documentation is well-written but it lacks the clear structure. I would suggest either to change the structure (I have some ideas) or to add more hyperlinks, like I have done for UI modes.

### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/manual/basics/creating_an_annotation_task.md [hunk 1]
```diff
@@ -61,11 +61,14 @@ There are two ways of working with labels (available only if the task is not rel
...
-  Click the `Continue` button to add more labels.
+  Click the `Continue` button to add more labels. You would immediately be able to observe the added labels in the `Raw` mode.
   If you need to cancel adding a label - press the `Cancel` button.
   After all the necessary labels are added click the `Done` button.
   After clicking `Done` the added labels would be displayed as separate elements of different colour.
-  You can edit or delete labels by clicking `Update attributes` or `Delete label`.
+  > Notice that you cannot observe the added labels in the `Constructor` mode when you create an annotation task. 
+  > You can do this only when you open [an existing task](/docs/manual/basics/task-details).
+  > There, you also can edit or delete labels by clicking `Update attributes` or `Delete label` in existing task.
+  > While you are creating a new task you can modify labels only in the `Raw` mode.
```
Comment (nmanovic) [inline site/content/en/docs/manual/basics/creating_an_annotation_task.md:71]: > While you are creating a new task you can modify labels only in the `Raw` mode.

Could you please explain why you think so?

File: site/content/en/docs/manual/basics/task-details.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ and the details of the task (specified when the task was created) and the jobs s
   - Duration — is the amount of time the job is being worked.
   - Assignee is the user who is working on the job.
     You can start typing an assignee’s name and/or choose the right person out of the dropdown list.
+    If you have not chosen any [organization] (/docs/manual/advanced/organization) or you have not chosen the assignee, the job is assigned to you.
```
Comment (nmanovic) [inline site/content/en/docs/manual/basics/task-details.md:46]: Could you please explain what you mean here?

Comment (ElinaAndreeva) [reply site/content/en/docs/manual/basics/task-details.md:46]: Well, my user case was: 

1. I sign up.
2. I do not choose or create any organization.
3. I create a new task.
4. I go to ‘Jobs’ assigned to me. The task is there.

File: site/content/en/docs/manual/basics/creating_an_annotation_task.md (hunk 1 above)
Comment (ElinaAndreeva) [reply site/content/en/docs/manual/basics/creating_an_annotation_task.md:71]: Something may not work correctly on my laptop but it was surprising for me too. When I create (not edit) a task, I add a label. Then, I click ‘Continue’, and I can no longer see it. When I go to ‘Raw’, all created labels are there.

File: site/content/en/docs/manual/basics/top-panel.md [hunk 3]
```diff
@@ -128,6 +128,10 @@ and type of annotation (shape, track). As well as the number of manual and inter
 
 ## UI switcher
 
-Switching between user interface modes.
-
+Switch between user interface modes.
+There are four available mode:
```
[the remark this answers is not in the document]
Comment (ElinaAndreeva) [reply site/content/en/docs/manual/basics/top-panel.md:132]: Yes, sorry, a typo.

**Pull request closed without merging**